tests/unit/_test_collision_analysis.py

```diff
@@ -2,9 +2,11 @@
     class_collision_in_columns, class_collision)
 
 
+# TODO: where is the assertion?
```

Comment:
didn't find a way to do an assertion on a display. Function is not returning anything is just displaying plots

Reply:
I guess this is one example where testing does not bring much-added value. 
Such cases, however, shall not be too many :)